knowledge_repo/app/static/css/custom.css

```diff
@@ -339,53 +346,116 @@ img {
   font-size: 18px;
 }
 
+#renderedMarkdown {
+    border: 1px solid #bbb;
+    padding: 1.5em;
+    background: #fff;
+    color: #484848;
+    margin-top: 2em;
+    margin-bottom: 2em;
+    position: relative;
+}
+
 #renderedMarkdown h1,
 #renderedMarkdown h2,
 #renderedMarkdown h3,
 #renderedMarkdown h4,
 #renderedMarkdown h5,
 #renderedMarkdown h6 {
-  margin-top: 50px;
-  margin-bottom: 25px;
+  margin-top: 1.5em;
```

Comment:
em means the margins are relative to font sizes, and if the user changes the font sizes, this will lead to more consistent padding. It also means that we can change the font-sizes in the future, and the spacing of headers should not need to be further adjusted to look sensible.